pyanaconda/modules/boss/installation.py

```diff
@@ -135,7 +135,7 @@ def _chmod_logs(self):
         """Set access bits"""
         items = glob.glob(join_paths(self._sysroot, ANACONDA_LOG_DIR, '*'))
         for item in items:
-            os.chmod(item, 0x0600)
+            os.chmod(item, 0o0600)
```

Comment:
It looks like it would be useful to have this in a separate function that can handle and log the exception.